Documentation/README.md

```diff
@@ -0,0 +1,39 @@
+# Microsoft Azure SDK for .NET Documentation
+
+The following services use AutoRest
+- AppServices
+- Compute
+- DataLake Analytics
+- DataLake Store
+- DataLake Storeuploader
+- DevTestLabs
+- Identity
+- Machine Learning
+- Networking
+- NotificationHubs
+- PowerBI
+- Redis Cache
+- Resource Manager (ARM)
+
+The following services use Hyak
+- Azure Backup
+- Azure Stack
+- Data Factory
+- Datalake AnalyticsCatalog
+- Datalake AnalyticsJob
+- DataLake StoreFileSystem
```

Comment:
3 DataLake services are deprecated. They have been moved to Swagger/Autorest and renamed to DataLake Analytics and DataLake Store